packages/client/ui/Dialog/SimpleModalDialog.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import * as RadixDialog from '@radix-ui/react-dialog'
+import {DialogOverlay} from './DialogOverlay'
+
+interface DialogProps {
+  isOpen: boolean
+  onClose: () => void
+  children: React.ReactNode
+  rootProps?: React.ComponentPropsWithoutRef<typeof RadixDialog.Root>
+  overlayProps?: React.ComponentPropsWithoutRef<typeof DialogOverlay>
+  portalProps?: React.ComponentPropsWithoutRef<typeof RadixDialog.Portal>
+}
+
+export const SimpleModalDialog = (props: DialogProps) => {
```

Comment:
+1 this could be removed if you take my suggestion from previous comment. and with that in mind, there'll be no need for props like `rootProps`, `overlayProps`, `portalProps` and these components will be directly accessible using composition